python/taichi/lang/__init__.py

```diff
@@ -113,8 +116,8 @@ def print_kernel_profile_info(mode='count'):
     """Print the profiling results of Taichi kernels.
 
     To enable this profiler, set ``kernel_profiler=True`` in ``ti.init()``.
-    The default print mode is ``COUNT`` mode: print the statistical results (min,max,avg time) of Taichi kernels,
-    another mode ``TRACE``: print the records of launched Taichi kernels with specific profiling metrics (time, memory load/store and core utilization etc.)
+    The default print mode is ``COUNT`` mode: print the statistics (min,max,avg time) of launched kernels,
```

Comment:
Again, let's not repeatedly write the same comments. They should basically be treated the same way as code, and it's better to have a single place to explain these :-)